dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/DockerfileGitHubUtil.java

```diff
@@ -80,7 +80,22 @@ public GHRepository getRepo(String repoName) throws IOException {
         if (query.substring(query.lastIndexOf(' ') + 1).length() <= 1) {
             throw new IOException("Invalid image name.");
         }
-        search.q("\"FROM " + query + "\"");
+        // Github search returns no results if your verbatim string contains more than two slashes
```

Comment:
I think I've seen these  in our environments. Perhaps we can add an image of the type that is not handled to the itest? Either way that'll help keep this supported to have that itest.

Reply:
Yeah, note the nuances here: https://github.com/salesforce/dockerfile-image-update/pull/191#pullrequestreview-577611089

We'll need to get the itests working again. They're currently ignored since they're failing in your org - was looking to migrate them to https://github.com/dockerfile-image-update.